sklearn/neighbors/tests/test_nearest_centroid.py

```diff
@@ -97,6 +97,20 @@ def test_pickle():
                        " after pickling (classification).")
 
 
+def test_shrinkage():
+    # Ensure that the shrinking is correct.
+    # The corrent result is calculated by R (pamr),
+    # which is implemented by the author of the original paper.
+    # (One need to modify the code to output the new centroid in pamr.predict)
```

Comment:
@jnothman Thanks. This is a regression test. But since it ensures that the shrinking is correct, I choose not to mention the issue number here.